docs/dictionary/command/iphoneSetFileDataProtection.lcdoc

```diff
@@ -21,26 +21,27 @@ iphoneSetFileDataProtection specialFolderPath("documents") & "/appointments.txt"
 
 Parameters:
 filename:
-The fulle path to the file to be protected.
+The full path to the file to be protected.
 
 dataProtection (enum):
 The level of protection to apply to the file
 
 -   "none": No protection
--   "complete": The file is not accessible, for read or write, while the
-    device is locked.
--   "complete unless open": The file is fully protected when the device
-    is locked, unless it was already open
--   "complete until first user authentication": The file is fully
-    protected until the user unlocks the device for this first time.
+-   "complete": The file is not accessible, for read or write, while the 
+device is locked.
+-   "complete unless open": The file is fully protected when the device 
+is locked, unless it was already open
+-   "complete until first user authentication": The file is fully 
+protected until the user unlocks the device for this first time.
```

Comment:
These enum descriptions will have to remain unwrapped until we fix http://quality.livecode.com/show_bug.cgi?id=18657